07.ls_object/ls.rb

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+require_relative 'lib/screen'
+require_relative 'lib/ls_file_stat'
+
+def main
+  all_visible = false
+  reverse = false
+  long_format = false
+
+  opt = OptionParser.new
+  opt.on('-a') { all_visible = true }
+  opt.on('-r') { reverse = true }
+  opt.on('-l') { long_format = true }
+
+  paths = opt.parse(ARGV)
+  paths = ['.'] if paths.empty?
```

Comment:
```suggestion
  paths << '.' if paths.empty?
```

みたいにして再代入を避けるといいかも
https://bootcamp.fjord.jp/pages/avoid-re-assign

Reply:
そのように変更しました！